tests/unit/sagemaker/jumpstart/test_notebook_utils.py

```diff
@@ -547,12 +541,15 @@ def vulnerable_training_model_spec(bucket, key, *args, **kwargs):
         patched_read_s3_file.side_effect = vulnerable_inference_model_spec
 
         num_specs = len(PROTOTYPICAL_MODEL_SPECS_DICT)
-        num_prop_specs = len(BASE_PROPRIETARY_MANIFEST)
         assert [] == list_jumpstart_models(
-            And("inference_vulnerable is false", "training_vulnerable is false")
+            And(
+                "inference_vulnerable is false",
+                "training_vulnerable is false",
+                "model_type is open_weights",
```

Comment:
do we need to include "model_type is open_weights" or is it still included by default?

Reply:
It is not included by default by design, `list_jumpstart_models` has both open weights and proprietary models for discoverability, and `"model_type is open_weights"` filter is used to only list open weights models.